api/components/restaurant/opening-hours.json

```diff
@@ -2,7 +2,7 @@
   "connection": "default",
   "collectionName": "components_opening_hours",
   "info": {
-    "name": "Opening_Hours",
+    "name": "opening-hours",
```

Comment:
Hey all good for me. Just want to make sure that this change won't break anything in the frontend. Could you confirm that?

Reply:
Hello @malgamves !
According to the [Model options](https://strapi.io/documentation/v3.x/concepts/models.html#model-information) documentation, the `name` key/value is used to display the name in the admin panel.
I update I did is to match the file format `opening-hours.json` with a `-` instead of a `_`.
And it breaks nothing in the frontend app, I double-checked.